index.html

```diff
@@ -2,357 +2,392 @@
-          <ul class="work-langs">
-            <li>
-              <p>html</p>
-            </li>
-            <li>
-              <p>css</p>
-            </li>
-            <li>
-              <p>javaScript</p>
-            </li>
-          </ul>
-          <button type="button" class="see-more">See Project</button>
-        </div>
-      </div>
-      <div class="work">
-        <img alt="work2" class="work-image" src="/assets/work2.png" />
-        <div>
-          <div>
-            <h2 class="work-title">Multi-Post Stories</h2>
-            <ul class="work-details">
-              <li>CANOPY</li>
-              <li>
-                <svg width="8" height="8" viewBox="0 0 8 8" fill="none" xmlns="http://www.w3.org/2000/svg">
-                  <circle cx="4" cy="4" r="4" fill="#C1C7D0" />
-                </svg>
-              </li>
-              <li>Back End Dev</li>
-              <li>
-                <svg width="8" height="8" viewBox="0 0 8 8" fill="none" xmlns="http://www.w3.org/2000/svg">
-                  <circle cx="4" cy="4" r="4" fill="#C1C7D0" />
-                </svg>
-              </li>
-              <li>2015</li>
-            </ul>
-          </div>
-          <p class="work-des">A daily selection of privately personalized reads; no accounts or sign-ups
-            required.</p>
-          <ul class="work-langs">
-            <li>
-              <p>html</p>
-            </li>
-            <li>
-              <p>css</p>
-            </li>
-            <li>
-              <p>javaScript</p>
-            </li>
-          </ul>
-          <button type="button" class="see-more">See Project</button>
-        </div>
-      </div>
-      <div class="work">
-        <img alt="work3" class="work-image" src="/assets/work3.png" />
-        <div>
-          <div>
-            <h2 class="work-title">Tonic</h2>
-            <ul class="work-details">
-              <li>CANOPY</li>
-              <li>
-                <svg width="8" height="8" viewBox="0 0 8 8" fill="none" xmlns="http://www.w3.org/2000/svg">
-                  <circle cx="4" cy="4" r="4" fill="#C1C7D0" />
-                </svg>
-              </li>
-              <li>Back End Dev</li>
-              <li>
-                <svg width="8" height="8" viewBox="0 0 8 8" fill="none" xmlns="http://www.w3.org/2000/svg">
-                  <circle cx="4" cy="4" r="4" fill="#C1C7D0" />
-                </svg>
-              </li>
-              <li>2015</li>
-            </ul>
-          </div>
-          <p class="work-des">A daily selection of privately personalized reads; no accounts or sign-ups
-            required.</p>
-          <ul class="work-langs">
-            <li>
-              <p>html</p>
-            </li>
-            <li>
-              <p>css</p>
-            </li>
-            <li>
-              <p>javaScript</p>
-            </li>
-          </ul>
-          <button type="button" class="see-more">See Project</button>
+                <ul class="desk-menu">
```

Comment:
- [ ] You well-implemented the requested changes by the previous reviewer. While implementing the requested changes I can see you introduced some styles issues. Please use the correct font-weight for navbar menu options and for all headings (Hero section, Projects section, about section) And make sure it matches the design on the [Figma ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)template.

### Your design

![image](https://user-images.githubusercontent.com/83803414/160239797-de8ffbd5-3504-4a6a-9da3-6fae14743245.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/160239805-e5f7c760-d931-4b93-a782-81de224767c7.png)

### Your design

![image](https://user-images.githubusercontent.com/83803414/160239821-0f2b721f-759d-4b69-ad34-959a3ddfe298.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/160239827-88188335-fab1-4452-9260-b34a13d889ef.png)

### Your design

![image](https://user-images.githubusercontent.com/83803414/160239865-ea72b67e-78bb-4c87-b0b3-1fa3e2266515.png)


### Figma design

![image](https://user-images.githubusercontent.com/83803414/160239872-7043abcb-0863-4279-b607-ab7a05bda6bf.png)